api/v1/cluster_types.go

```diff
@@ -99,8 +99,11 @@ type WorkerGroupSpec struct {
 }
 
 type ModelCache struct {
-	ModelRegistryType ModelRegistryType            `json:"model_registry_type,omitempty" yaml:"model_registry_type,omitempty"`
-	HostPath          *corev1.HostPathVolumeSource `json:"host_path,omitempty" yaml:"host_path,omitempty"`
+	// Deprecated: Model caching is registry-independent, so ModelRegistryType is no longer used.
+	ModelRegistryType ModelRegistryType `json:"model_registry_type,omitempty" yaml:"model_registry_type,omitempty"`
```

Comment:
Shall we entirely remove it at the DB level, since v1.0 is a good timing for breaking changes.